Docker login steps are missing for exposed registry with both secured and insecured

tpoitras opened the pull request:
incorp closed #2710, and updated based on review from Jaspreet Kaur (comment9 in bz): https://bugzilla.redhat.com/show_bug.cgi?id=1360586#c9

tpoitras commented:
Nicely rendered docs here.

Added this: http://file.bne.redhat.com/tpoitras/2016/registry/openshift-enterprise/registry-login-2-BZ1360586/install_config/registry/securing_and_exposing_registry.html#access-secure-registry-by-exposing-route

And this: http://file.bne.redhat.com/tpoitras/2016/registry/openshift-enterprise/registry-login-2-BZ1360586/install_config/registry/securing_and_exposing_registry.html#access-insecure-registry-by-exposing-route

tpoitras commented:
currently ON QA

tpoitras commented on install_config/registry/securing_and_exposing_registry.adoc:
```diff
@@ -293,3 +293,92 @@ not the route name and port. See `oc get imagestreams` for details.
+After securing and exposing the registry, update your *_/etc/sysconfig/docker_* file by adding the port number to the `OPTIONS` entry. For example:
+
+----
+OPTIONS='--selinux-enabled --insecure-registry=172.30.0.0/16 --insecure-registry registry.ose-node.openshift.com:80'
```
@bmeng can I just use any other port in the example? Like port 90?

bmeng replied:
The secured route will use the 443 port by default.
BTW, seems if we secured the docker registry, the option `--insecure-registry registry.ose-node.openshift.com:80` is not needed in the docker daemon config `/etc/sysconfig/docker`.

tpoitras replied:
@bmeng - I'll remove the part about updating that file, then. Will omitting it be enough, or do you think we should tell users to remove `--insecure-registry` entries from the `/etc/sysconfig/docker` file?

bmeng replied:
I think it is ok to just remove this part, since the `--insecure-registry` should not have such value by default.

tpoitras commented on install_config/registry/securing_and_exposing_registry.adoc:
```diff
@@ -293,3 +293,92 @@ not the route name and port. See `oc get imagestreams` for details.
+registry, make sure you specify the registry in the login command. For example:
+
+----
+docker login -e user@company.com -u f83j5h6 -p Ju1PeM47R0B92Lk3AZp-bWJSck2F7aGCiZ66aFGZrs2 registry.ose-node.openshift.com:80
```
@bmeng can I just use any other port in the example? Like port 90?

bmeng replied:
The default port for the secured route should be 443.

tpoitras commented:
Thank you, @bmeng 
I made your changes, but just guessed at a port number. Do you have a suggested port number I should use for the secured registry example commands?

tpoitras commented:
Updated based on latest suggestions from @bmeng 
Moving along to peer review: @bfallonf @ahardin-rh @adellape 

Nicely rendered docs:

Added this: http://file.bne.redhat.com/tpoitras/2016/registry/openshift-enterprise/registry-login-2-BZ1360586/install_config/registry/securing_and_exposing_registry.html#access-secure-registry-by-exposing-route

And this: http://file.bne.redhat.com/tpoitras/2016/registry/openshift-enterprise/registry-login-2-BZ1360586/install_config/registry/securing_and_exposing_registry.html#access-insecure-registry-by-exposing-route

ahardin-rh commented:
@tpoitras LGTM!

tpoitras commented:
Thanks @ahardin-rh , merging! :neckbeard:

tpoitras commented:
[rev_history]
|xref:../install_config/registry/securing_and_exposing_registry.adoc#install-config-registry-securing-exposing[Installation and Configuration -> Setting up the Registry -> Securing and Exposing the Registry]
|Added two new sections on Exposing a Secure Registry and Exposing a Non-Secure Registry. 
%
